packages/farrow-api/src/generateApi.ts

```diff
@@ -123,12 +123,25 @@ const getFieldsType = (fields: FormatFields, types: FormatTypes): string[] => {
   })
 }
 
+export type ApiClientHelpers = {
+  importStatements: (formatResult: FormatResult, options?: CodegenOptions) => string[]
+  typeDeclarations: (formatResult: FormatResult, options?: CodegenOptions) => string[]
+  variableDeclarations: (formatResult: FormatResult, options?: CodegenOptions) => string[]
+  apiFunctionParams: (
+    formApi: FormatApi,
```

Comment:
Is `formApi` short for `formatApi`? It confuses.

Reply:
my mistake, it is a typo